test/datatable.jl

```diff
@@ -341,4 +341,26 @@ module TestDataTable
     @test find(c -> isa(c, NullableCategoricalArray), categorical!(DataTable(A=1:3, B=4:6), :A).columns) == [1]
     @test find(c -> isa(c, NullableCategoricalArray), categorical!(DataTable(A=1:3, B=4:6), [1]).columns) == [1]
     @test find(c -> isa(c, NullableCategoricalArray), categorical!(DataTable(A=1:3, B=4:6), 1).columns) == [1]
+
+    @testset "duplicate entries in unstack warnings" begin
+        dt = DataTable(id=NullableArray([1, 2, 1, 2]), variable=["a", "b", "a", "b"], value=[3, 4, 5, 6])
+        if isdefined(Symbol(:@test_warn))
```

Comment:
Should be `isdefined(Base.Test, Symbol("@test_warn"))`. This is why the tests failed.